packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -163,6 +164,8 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
         clientSecret,
         silenceWarnings,
         configLogger,
+        defaultDnt,
+        callbackURI,
```

Comment:
this was needed so that the callbackURI could be picked up automatically

Reply:
Is this callback URI specially used for passwordless?Can it be used for something else?